moveit_core/robot_model/include/moveit/robot_model/robot_model.h

```diff
@@ -37,21 +37,24 @@
 
 #pragma once
 
-#include <moveit/macros/class_forward.h>
-#include <moveit/exceptions/exceptions.h>
 #include <urdf/model.h>
+
 #include <srdfdom/model.h>
 
+#include <moveit/exceptions/exceptions.h>
+#include <moveit/macros/class_forward.h>
+
 // joint types
-#include <moveit/robot_model/joint_model_group.h>
+#include <iostream>
+
+#include <Eigen/Geometry>
+
 #include <moveit/robot_model/fixed_joint_model.h>
 #include <moveit/robot_model/floating_joint_model.h>
+#include <moveit/robot_model/joint_model_group.h>
 #include <moveit/robot_model/planar_joint_model.h>
-#include <moveit/robot_model/revolute_joint_model.h>
 #include <moveit/robot_model/prismatic_joint_model.h>
-
-#include <Eigen/Geometry>
-#include <iostream>
+#include <moveit/robot_model/revolute_joint_model.h>
```

Comment:
One of the challenging things about this PR is that comments above blocks of includes essentially breaks the re-rodering.  Here it re-ordered above and below the `// joint types` line as separate blocks.  I've gone through and removed a bunch of these comments but here is one I forgot.